packages/instrumenter/src/transformers/directive-bookkeeper.ts

```diff
@@ -89,4 +96,21 @@ export class DirectiveBookkeeper {
     mutatorName = mutatorName.toLowerCase();
     return this.currentIgnoreRule.findIgnoreReason(mutatorName, line);
   }
+
+  private warnAboutUnusedDirective(mutators: string, directiveType: string, scope: string) {
+    const directives = mutators
+      .split(',')
+      .map((mutator) => mutator.trim())
+      .filter((mutator) => mutator !== WILDCARD);
+    for (const directive of directives) {
+      if (!this.allMutators.map((x) => x.name.toLowerCase()).includes(directive.toLowerCase())) {
```

Comment:
Can we do this `map` in the constructor of this class? Now we're doing it every time we find a directive. 

Its a small performance hit, but mostly this `warn` function shouldn't be doing too much logic.